packages/@react-aria/menu/src/useMenuItem.ts

```diff
@@ -85,7 +83,13 @@ export interface AriaMenuItemProps {
   onAction?: (key: Key) => void,
 
   /** What kind of popup the item opens. */
-  'aria-haspopup'?: 'menu' | 'dialog'
+  'aria-haspopup'?: 'menu' | 'dialog',
+
+  /** Indicates whether the menu item's popup element is expanded or collapsed. */
+  'aria-expanded'?: boolean | 'true' | 'false',
```

Comment:
Looks like the docs have a hard time expanding React's type for `aria-expanded` (doesn't show Booleanish in the prop table). I could make our own booleanish type, will dig further in case your react types PR helps the docs process this stuff